### PR TITLE
New component about me and fix border styles

### PR DESCRIPTION
**Pull request by dmytro-slobodian-wttech**
merge border and border color
new component About Me

### CODE REVIEW

File: src/components/SearchBlock.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  </div>
+</template>
+
+<style scoped>
+.searchblock_form_button {
+  cursor: pointer;
+  font-size: var(--size-blog-main-text);
+  position: relative;
+  margin-left: -35px;
+  border: 0px;
```
Comment (thisisui) [inline src/components/SearchBlock.vue]: 0 should be without 'px'

File: src/components/SearchBlock.vue [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  background-color: var(--main-bgdiv-color);
+  padding-right: 0px;
```
Comment (thisisui) [inline src/components/SearchBlock.vue]: 0 without unit please

**Review by thisisui — CHANGES_REQUESTED** [review]

File: src/components/Postblock.vue [hunk 3]
```diff
@@ -0,0 +1,141 @@
+<script setup></script>
+
+<template>
+  <div class="top_image_post_block">
+    <img
+      class="top_image_post_block_Img_main"
```
Comment (igor-napierala-wttech) [inline src/components/Postblock.vue:6]: Img should be img

File: src/components/Postblock.vue [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  padding-top: 25px;
+  margin-left: auto;
+  margin-right: auto;
+  color: var(--text-color-light);
+  font-size: var(--size-blog-info-text);
+  width: 80%;
+}
+
+.top_image_post_block_info_block_name {
+  margin-top: 20px;
```
Comment (igor-napierala-wttech) [inline src/components/Postblock.vue:74]: you can mix 3 lines in one 

`margin: 20px auto inherit;`

probably it was initial or inherit - please check in docs https://developer.mozilla.org/pl/docs/Web/CSS/Margin

File: src/components/Postblock.vue [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+  margin-left: auto;
+  margin-right: auto;
+  color: var(--text-color-darkness);
+  font-size: var(--size-blog-name-text);
+  font-weight: 600;
+  width: 80%;
+}
+
+.top_image_post_block_info_block_date {
+  margin-top: 20px;
```
Comment (igor-napierala-wttech) [inline src/components/Postblock.vue:84]: merge 3 lines into one - margin ones

File: src/components/Postblock.vue [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+  margin-left: auto;
+  margin-right: auto;
+  color: var(--text-color-lightness);
+  font-size: var(--size-blog-info-text);
+  width: 80%;
+}
+
+.top_image_post_block_info_block_mainText {
+  margin-top: 20px;
```
Comment (igor-napierala-wttech) [inline src/components/Postblock.vue:93]: merge 3 lines into one - margin ones

**Pull request merged**